source/translator.d

```diff
@@ -0,0 +1,35 @@
+private string[string] defaultLanguage;
+static this()
+{
+	defaultLanguage = [
+		"editor.run": "Run",
+		"editor.reset": "Reset",
+		"editor.format": "Format",
+		"editor.keyboard_shortcuts": "Keyboard Shortcuts",
+	];
+}
+
+/**
+Allows per-language overwrites of UI text
+*/
+class Translator
```

Comment:
But hashtable inside is already a reference type, so copying the struct means copying the reference to the hashtable ;)
I don't feel too strongly about this so we can leave it be.